app/controllers/users_controller.rb

```diff
@@ -170,7 +170,18 @@ def changed_email
     if !params[:new_email].blank? && reauthenticate
       new_email = params[:new_email]
 
-      if new_email != params[:email_confirmation]
+      # Please note: This comparison is not technically
```

Comment:
You can wrap your comments at around 80 characters, like a few other places in this file. I think this is a bit narrow.

Thanks for looking into this! I didn't know that the local part of emails is supposed to be case sensitive. The pragmatic choice makes sense.